docs/source/emb_sim.rst

```diff
@@ -0,0 +1,99 @@
+dglke_emb_sim: finding similar embeddings
+-------------------------------------------
+This task is to find the most similar entity/relation embeddings for some pre-defined similarity functions given a set of entities or relations. An example of the output for top5 similar entities are as follows::
+
+    left     right    score
+    0        0        0.99999
+    0        18470    0.91855
+    0        2105     0.89916
+    0        13605    0.83187
+    0        36762    0.76978
+
+Currently we support five different similarity functions: cosine, l2 distance, l1 distance, dot product and extended jaccard.
+
+Four arguments are required to provide basic information for finding similar embeddings:
+
+  * ``--emb_file``, The numpy file that contains the embeddings of all entities/relations in a knowledge graph.
+  * ``--format``, The format of the input objects (entities/relations).
+
+    * ``l_r``: two list of objects are provided as left objects and right objects.
+    * ``l_*``: one list of objects is provided as left objects and all objects in emb\_file are right objects. This is to find most similar objects to the ones on the left.
+    * ``*_r``: one list of objects is provided as right objects list and treat all objects in emb\_file as left objects.
```

Comment:
what is the difference of `l_*` and `*_r`? their function should be exactly the same.

Reply:
For the exec_mode, we only support pairwise, all and batch_left. So l_* and *_r is different in batch_left.